components/commander/common/commander_url.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/commander/common/commander_url.h"
+
+#include <string>
+#include <vector>
+
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/string_split.h"
+#include "base/strings/string_util.h"
+#include "base/unguessable_token.h"
+
+namespace commander {
+namespace {
+constexpr char kBraveCommandsScheme[] = "brave-commands";
+}  // namespace
+
+GURL GetCommandURL(uint32_t command_index, uint32_t result_set_id) {
+  return GURL(kBraveCommandsScheme + std::string("://") +
```

Comment:
for defense in depth, it might be good to keep that unguessable token in here unless you're 100% sure that nothing happens if a command URL is opened in Brave from an external app or website.

Reply:
Definitely seems like it can't hurt!